capa/v29.0.0/release.yaml

```
@@ -0,0 +1,120 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: aws-29.0.0
+spec:
+  apps:
+  - name: aws-ebs-csi-driver
+    version: 2.30.1
+    dependsOn:
+    - cloud-provider-aws
+  - name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0
+    dependsOn:
+    - cert-manager
+  - name: aws-pod-identity-webhook
+    version: 1.16.0
+    dependsOn:
+    - cert-manager
+  - name: capi-node-labeler
+    version: 0.5.0
+  - name: cert-exporter
+    version: 2.9.0
+    dependsOn:
+    - kyverno
+  - name: cert-manager
+    version: 3.7.6
+    dependsOn:
+    - prometheus-operator-crd
+  - name: chart-operator-extensions
+    version: 1.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cilium
+    version: 0.24.0
```

```suggestion
    version: 0.25.1
```

Please upgrade cilium-app to 0.25.1 (CAPA 25.1.0 will also have this version)
The changes between 0.24.0 and 0.25.1 are just a cilium patch bump to 1.15.6 and said BPF map size regression

thanks

---

https://github.com/giantswarm/releases/pull/1323

I created that PR including all Cabbage apps

---

Thank you, Mati! This is how we had it in mind!